bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/services/sitemap/CategoriesSitemapGenerator.java

```diff
@@ -178,19 +174,25 @@ public void generate(Resource sitemapRoot, String name, Sitemap sitemap, Context
         }
     }
 
-    private static QueryQueryDefinition categoryQueryFor(String categoryUid) {
+    private QueryQueryDefinition categoryQueryFor(String categoryUid) {
         return q -> q.categories(
             arguments -> arguments
                 .filters(new CategoryFilterInput()
                     .setCategoryUid(new FilterEqualTypeInput()
                         .setEq(categoryUid))),
             resultSet -> resultSet
-                .items(category -> category
-                    .urlKey()
-                    .urlPath()
-                    .uid()
-                    .createdAt()
-                    .updatedAt()
-                    .children(child -> child.uid())));
+                .items(category -> {
+                    category
+                        .urlKey()
+                        .urlPath()
+                        .uid()
+                        .children(child -> child.uid());
+
+                    if (addLastModified) {
```

Comment:
Those fields are deprecated in the GraphQL API btw.

Reply:
This change is unrelated to the ticket ... I missed that.

However we managed to convince the Magento team to keep those fields and un-deprecate :)